Exiled.API/Features/Items/Firearm.cs

```diff
@@ -138,9 +156,10 @@ public byte Ammo
         public bool Aiming => Base.AdsModule.ServerAds;
 
         /// <summary>
-        /// Gets a value indicating whether the firearm's flashlight module is enabled.
+        /// Gets a value indicating whether the firearm's flashlight module is enabled or if playing see trough a NightVision.
         /// </summary>
-        public bool FlashlightEnabled => Base.Status.Flags.HasFlagFast(FirearmStatusFlags.FlashlightEnabled);
+        public bool FlashlightEnabled => Base.Status.Flags.HasFlagFast(FirearmStatusFlags.FlashlightEnabled) ||
```

Comment:
Separate property for night vision

Reply:
How would you call the property that combined both ?

Reply:
Separate property for Flashlight vs night vision

Reply:
> Separate property for Flashlight vs night vision

I still want a property than combine both of them how should i call it ?

Reply:
IsEmittingLight would probably work